client/src/components/AppHeader.vue

```diff
@@ -4,14 +4,16 @@
     <app-banner />
     <q-toolbar class="header-toolbar">
       <div class="q-pa-sm row q-gutter-md items-center">
-        <q-img
+        <a href="/"><q-img
           :alt="$t('header.logo_alt')"
           src="/logo-100x100.png"
           style="width: 50px; height: 50px"
-        />
+        /></a>
         <div v-if="$q.screen.width >= 400" class="column">
-          <h1 class="q-ma-none text-h4 site-title">Pilcrow</h1>
-          <small class="site-subtitle">Submission Review System</small>
+          <h1 class="q-ma-none text-h4 site-title">
+            {{ $t("header.site_title") }}
+          </h1>
+          <small class="site-subtitle">{{ $t("header.site_subtitle") }}</small>
         </div>
       </div>
```

Comment:
Given the amount of segmentation in the header and the pattern established by the elements on the right side of the header, I think users will expect a broadly clickable area around both the logo and the text beside it. 

To make this work, I suggest replacing lines 6–18 with this. I'd offer this as an official code suggestion but GitHub says, "applying suggestions on deleted lines is not supported." 

```vue
      <router-link
        to="/"
        class="q-pa-sm row q-gutter-md items-center text-white"
        style="text-decoration: unset"
      >
        <q-img
          :alt="$t('header.logo_alt')"
          src="/logo-100x100.png"
          style="width: 50px; height: 50px"
        />
        <div v-if="$q.screen.width >= 430" class="column">
          <h1 class="q-ma-none text-h4 site-title">
            {{ $t("header.site_title") }}
          </h1>
          <small class="site-subtitle">{{ $t("header.site_subtitle") }}</small>
        </div>
      </router-link>
```